src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -89,11 +89,13 @@ public final class JavadocTokenTypes {
      * <pre>{@code @since 3.4 RELEASE}</pre>
      * <b>Tree:</b>
      * <pre>{@code
-     *   |--JAVADOC_TAG[3x0] : [@since 3.4 RELEASE]
-     *       |--SINCE_LITERAL[3x0] : [@since]
-     *       |--WS[3x6] : [ ]
-     *       |--DESCRIPTION[3x7] : [3.4 RELEASE]
-     *           |--TEXT[3x7] : [3.4 RELEASE]
+     * JAVADOC_TAG -> JAVADOC_TAG
+     *  |--SINCE_LITERAL -> @since
+     *  |--WS ->
+     *  `--DESCRIPTION -> DESCRIPTION
+     *      |--TEXT -> 3.4 RELEASE
+     *      |--NEWLINE -> \r\n
+     *      `--TEXT ->
```

Comment:
@SteLeo1603 ,
The codeblock shown in example is just `@since 3.4 RELEASE`
Please remove lines 97 & 98.
The AST till line 96 is only what we need here

Reply:
@SteLeo1603 , please do.